NOLF2/Shared/WinUtil.cpp

```diff
@@ -73,14 +73,19 @@ BOOL CWinUtil::DirExist (char const* strPath)
 
 #ifndef __LINUX
 	UINT oldErrorMode = SetErrorMode (SEM_FAILCRITICALERRORS | SEM_NOOPENFILEERRORBOX);
-#endif
```

Comment:
please reorder this into just 2 sections
the Non-__LINUX and the other like the FileExists function